freeorion.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>freeorion.desktop</id>
+  <name>FreeOrion</name>
+  <summary>Turn-based space empire and galactic conquest (4X) computer game</summary>
+  <summary xml:lang="de">Rundenbasierendes Weltraum-Strategiespiel</summary>
+  <developer_name>FreeOrion Project</developer_name>
+  <description>
+      <p>
+       FreeOrion is a free, open source, turn-based space empire and galactic conquest (4X) computer game being designed and built by the FreeOrion project. FreeOrion is inspired by the tradition of the Master of Orion games, but is not a clone or remake of that series or any other game.
+      </p>
+  </description>
+  <keywords>
+    <keyword>game</keyword>
+    <keyword>strategy</keyword>
+  </keywords>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <url type="bugtracker">https://github.com/freeorion/freeorion/issues</url>
+  <url type="donation">http://www.freeorion.org/index.php/Donations</url>
+  <url type="faq">http://www.freeorion.org/index.php/FAQ</url>
+  <url type="help">http://www.freeorion.org/forum/viewforum.php?f=25</url>
+  <url type="homepage">http://www.freeorion.org</url>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Galaxy setup</caption>
+      <image height="600" width="786">http://www.freeorion.org/images/thumb/1/1a/Freeorion_0.4.5_b2015-09-01.f203162_singleplayer.JPG/786px-Freeorion_0.4.5_b2015-09-01.f203162_singleplayer.JPG</image>
+    </screenshot>
+    <screenshot>
+      <caption>Ship design</caption>
+      <image height="599" width="787">http://www.freeorion.org/images/thumb/c/c7/Freeorion_0.4.5_b2015-09-01.f203162_ingame-windows_8.JPG/787px-Freeorion_0.4.5_b2015-09-01.f203162_ingame-windows_8.JPG</image>
```

Comment:
Are those reliable permanent URLs?  Do we need to modify the infrastructure?

Reply:
Per [MediaWiki manual - thumb.php](https://www.mediawiki.org/wiki/Manual:Thumb.php) looks like this specific thumbnail should be:
`http://freeorion.org/thumb.php?f=Freeorion_0.4.5_b2015-09-01.f203162_ingame-windows_8.JPG&w=787`
If a generated thumbnail is saved may depend on configuration of LocalSettings.php

Infrastructure change does not look required, but may be desirable to change filenames. I've posted a response in forum [thread](http://freeorion.org/forum/viewtopic.php?f=4&t=10676)

Reply:
Can host a file directly instead of relying on wiki upload, if necessary.

Reply:
> Can host a file directly instead of relying on wiki upload, if necessary.

That's probably a better idea than to rely on some wikimedia internal URL, that may change in the next version.

Reply:
@geoffthemedio Any progress here?